pallets/farming/src/tests.rs

```diff
@@ -406,3 +423,28 @@ fn test() {
         assert_eq!(eve_reward, balance!(375576.575413164636883116));
     });
 }
+
+#[test]
+fn set_lp_min_xor_for_bonus_reward_should_forbid_for_non_root_call() {
+    ExtBuilder::default().build().execute_with(|| {
+        assert_noop!(
+            Pallet::<Runtime>::set_lp_min_xor_for_bonus_reward(
+                RuntimeOrigin::signed(ALICE()),
+                balance!(100000)
+            ),
+            BadOrigin
+        );
+    });
+}
+
+#[test]
+fn set_lp_min_xor_for_bonus_reward_should_work() {
+    ExtBuilder::default().build().execute_with(|| {
+        let modified_min_xor = balance!(3 * (10_i32.pow(6)));
+        assert_ok!(Pallet::<Runtime>::set_lp_min_xor_for_bonus_reward(
+            RawOrigin::Root.into(),
+            modified_min_xor
+        ));
+        assert!(Pallet::<Runtime>::lp_min_xor_for_bonus_reward() == modified_min_xor);
```

Comment:
It would be nice to test that the event was emitted with `assert_has_event`.

Reply:
added assert for event emitting.
@Alexey-N-Chernyshov